keps/sig-network/2433-topology-aware-hints/README.md

```diff
@@ -490,7 +532,7 @@ feature.
 
 #### Sample Events
 
-| Type | Reason | Message | 
+| Type | Reason | Message |
```

Comment:
Re-reviewing this (and the PRR) and some minor comments:

L605: and kube-apiserver, right (for the new API field that is also protected by FG)

L621: were the tests mentioned there added? if so, can you link them?

L645: Were the manual tests run? Can you describe the test?


You will probably also be asked to update to a new template (there is one additional new question in scalability section there).

Reply:
Thanks @wojtek-t, will add these.

Reply:
>L605: and kube-apiserver, right (for the new API field that is also protected by FG)

Added, thanks!

>L621: were the tests mentioned there added? if so, can you link them?

Added links to tests that exist, TODO for one that doesn't.

>L645: Were the manual tests run? Can you describe the test?

They might have been, but I can't remember, will make sure they're run + properly described here before 1.28 cycle.

>You will probably also be asked to update to a new template (there is one additional new question in scalability section there).

Sounds good, will catch this in the next cycle, now that we're waiting until at least 1.28 for GA.